tests/testing_common.py

```diff
@@ -665,8 +666,10 @@ def _test_safe_merge(self, model_id, config_cls, config_kwargs):
         model = get_peft_model(model, config).eval()
         logits_peft = model(**inputs)[0]
 
-        # sanity check that the logits are different
-        assert not torch.allclose(logits_base, logits_peft, atol=1e-6, rtol=1e-6)
+        # LNTuning do not change the logits
```

Comment:
To clarify a bit more:

```suggestion
        # Initializing with LN tuning cannot be configured to change the outputs (unlike init_lora_weights=False)
```